CHANGES.md

```diff
@@ -1,4 +1,192 @@
+    single-namespace mode, which was severely limited in the number of features
+    it supported due to not having access to cluster-wide resources
+  * Updated automatic proxy injection and CLI injection to support overriding
+    inject defaults via pod spec annotations
+  * Added a new public API endpoint for fetching control plane configuration
+  * Updated the `mutatingwebhookconfiguration` so that it is recreated when the
+    proxy injector is restarted, so that the MWC always picks up the latest
+    config template during version upgrade
+  * Updated destination service to return TLS identities only when the
+    destination pod is TLS-aware and is in the same controller namespace
+  * Lessen klog level to improve security
+  * Updated control-plane components to query Kubernetes at startup to determine
+    authorized namespaces and if ServiceProfile support is available
+  * Modified the stats payload to include the following TCP stats:
+    `tcp_open_connections`, `tcp_read_bytes_total`, `tcp_write_bytes_total`
+  * Instrumented clients in the control plane connecting to Kubernetes, thus
+    providing better visibility for diagnosing potential problems with those
+    connections
+  * Renamed the "linkerd-proxy-api" service to "linkerd-destination"
+  * Bumped Prometheus to version 2.7.1 and Grafana to version 5.4.3
+* Proxy
+  * **Fixed** a connection starvation issue where TLS discovery detection on
+    slow or idle connections could block all other connections from being
+    accepted on the inbound listener of the proxy
+  * Some `l5d-*` informational headers have been temporarily removed from
+    requests and responses because they could leak information to external
+    clients
+  * **Fixed** a stream leak between the proxy and the control plane that could
+    cause the `linkerd-controller` pod to use an excessive amount of memory
+  * Introduced per-proxy private key generation and dynamic certificate renewal
+  * Added a readiness check endpoint on `:4191/ready` so that Kubernetes doesn't
+    consider pods ready until they have acquired a certificate from the Identity
+    controller
+  * The proxy's connect timeouts have been updated, especially to improve
+    reconnect behavior between the proxy and the control plane
+  * Increased the inbound/router cap on MAX_CONCURRENT_STREAMS
+  * The `l5d-remote-ip` header is now set on inbound requests and outbound
+    responses
+  * Fixed issue with proxy falling back to filesystem polling due to improperly
+    sized inotify buffer
+* Web UI
+  * The Overview page in the Linkerd dashboard now renders appropriately when
+    viewed on mobile devices
+  * Removed TLS columns from the dashboard tables; this is in preparation for
+    surfacing identity metrics in a clearer way
+  * The topology graph now shows TCP stats if no HTTP stats are available
+  * The resource detail page no longer shows blank tables if the resource only
```

Comment:
tioli: maybe `Improved table display on the resource detail page for resources with TCP-only traffic`